GoogleContainerTools/jib
# Adds docs-unreleased/ for storing unreleased docs/.

How `docs/` and `docs-unreleased/` will be managed:

`docs/` stores latest released version documentation
`docs-unreleased/` stores docs for next release - during development, updates should be made to these docs

`docs/index.md` should have links to older versions’ docs at specific commits (eg. https://github.com/GoogleContainerTools/jib/tree/03d42652e82c543d0a6c6b69bd8827ec8effd82d for maven v0.1.7 and gradle v0.1.1)

Upon a new version release:
1. Delete `docs/` (except for `README.md`)
1. Copy `docs-unreleased/` to `docs/` (except for `README.md`)
